server/ntb/io/feeding_services/ntb_reuters_api.py

```diff
@@ -212,14 +231,19 @@ def get_query(self):
                     headLine
                     urgency
                     firstCreated
-                    subject {
+                    subject {{
                         name
                         code
-                    }
-                }
-            }
-        }
-    """
+                    }}
+                }}
+            }}
+        }}
+        """.format(
+            ", ".join(query_params.values()),
```

Comment:
can you use kwargs for those params so it's more obvious where they are inserted in the string?